src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/BeneficialOwnerChangeService.java

```diff
@@ -289,11 +288,21 @@ private BeneficialOwnerChange<OtherBeneficialOwnerPsc> convertBeneficialOwnerOth
 
     var beneficialOwnerNatureOfControlTypes = beneficialOwnerGovernmentOrPublicAuthorityDto.getBeneficialOwnerNatureOfControlTypes();
     var nonLegalFirmMembersNatureOfControlTypes = beneficialOwnerGovernmentOrPublicAuthorityDto.getNonLegalFirmMembersNatureOfControlTypes();
+    var trustControlNatureOfControlTypes = beneficialOwnerGovernmentOrPublicAuthorityDto.getTrustControlNatureOfControlTypes();
+    var ownerOfLandPersonNatureOfControlJurisdictions = beneficialOwnerGovernmentOrPublicAuthorityDto.getOwnerOfLandPersonNatureOfControlJurisdictions();
+    var ownerOfLandOtherEntityNatureOfControlJurisdictions = beneficialOwnerGovernmentOrPublicAuthorityDto.getOwnerOfLandOtherEntityNatureOfControlJurisdictions();
+    var nonLegalFirmControlNatureOfControlTypes = beneficialOwnerGovernmentOrPublicAuthorityDto.getNonLegalFirmControlNatureOfControlTypes();
+
 
     var naturesOfControlCollection =
             NaturesOfControlCollectionBuilder.createNaturesOfControlCollectionBuilder()
                     .addPersonType(beneficialOwnerNatureOfControlTypes)
                     .addFirmType(nonLegalFirmMembersNatureOfControlTypes)
+                    .addTrustType(trustControlNatureOfControlTypes)
```

Comment:
I'm happy to change that. This will be the last time this code is probably touched (maybe) so will look into it.

I might also need to refactor some tests now that @mattch1 has merged the Corporate BO changes.

Reply:
Happy for this name to change it would match what we have for the firm control type.

Reply:
I've changed the name of this now.